com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerCounterValue.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Unity.Collections.LowLevel.Unsafe;
+#if UNITY_2020_2_OR_NEWER
+using Unity.Profiling;
+using Unity.Profiling.LowLevel;
+using Unity.Profiling.LowLevel.Unsafe;
+#endif
+using UnityEngine;
+
+namespace MLAPI.Profiling
+{
+#if ENABLE_PROFILER
+    [StructLayout(LayoutKind.Sequential)]
+#else
+    [StructLayout(LayoutKind.Sequential, Size = 0)]
+#endif
+    public readonly struct ProfilerCounterValue<T> where T : unmanaged
```

Comment:
Nit, should be internal? Especially since this is sort of a copy of the upcoming work in the core package, we don't want anyone to start leveraging this

Reply:
Done